std/manual.md

```diff
@@ -1298,6 +1298,40 @@ All listeners added using `window.addEventListener` were run, but
 `window.onload` and `window.onunload` defined in `main.ts` overridden handlers
 defined in `imported.ts`.
 
+## `deno fmt`
+
+Deno ships with a built in code formatter that auto-formats TypeScript and
+JavaScript code.
+
+```shell
+# format all JS/TS files in the current directory and subdirectories
+deno fmt
+# format specific files
+deno fmt myfile1.ts myfile2.ts
+# check if all the JS/TS files in the current directory and subdirectories are formatted
+deno fmt --check
+# format stdin and write to stdout
+cat file.ts | deno fmt -
+```
+
+Ignore code to format by preceding it with a `// deno-fmt-ignore` comment:
+
+<!-- prettier-ignore-start -->
```

Comment:
One day I will finish the markdown formatter... (I at least have the JSON formatter in Rust now, but still a bit of work left on the markdown one).